### DISCUSSION TITLE
Need help compiling

### DISCUSSION THREAD

**Discussion by JustinKScott** [EdgeTX/edgetx#4094] [General]
Good Morning.  I'm having trouble with compilation; could someone push me in the right direction?

-- GITPOD --

A few weeks back, I used GITPOD successfully to create a build for my Taranis.  I wanted to make a few tweaks, but the GITPOD is no longer working.

Here is my config:
![Screenshot 2023-09-23 at 8 56 25 AM](https://github.com/EdgeTX/edgetx/assets/5257743/4974c222-dd02-4398-9084-ae58b95e64c9)

After a few minutes, this is what I get:

![Screenshot 2023-09-23 at 9 02 05 AM](https://github.com/EdgeTX/edgetx/assets/5257743/47158d63-c3bf-4627-85b7-f8aa1727079f)

Sometimes I get this:

![Screenshot 2023-09-23 at 8 57 28 AM](https://github.com/EdgeTX/edgetx/assets/5257743/0f1798c5-5b61-4466-8d58-bac451db1dc0)

So I abandoned hope with GITPOD, and went straight to MACOS

-- MacOS --

FYI  - I had a little trouble here:
>NJ108752LT:build jscott$ brew install sdl fox cmake
Warning: No available formula with the name "homebrew/core/sdl". Did you mean homeworlds?
Please tap it and then try again: brew tap homebrew/core

So I did:
>NJ108752LT:build jscott$ brew tap homebrew/core

... then everything seemed to work as expected.

Until I got to the make step:

![Screenshot 2023-09-23 at 9 03 05 AM](https://github.com/EdgeTX/edgetx/assets/5257743/44cfb118-3885-4608-8056-268ecbfa2243)

---

I would be most appreciative if someone could push me in the right direction?  
... getting to "Hello World" is always the most difficult step, you know?  :)

**Comment by gagarinlg** [comment]
Github changed something regarding gitpod, it will work again soon

**Comment by pfeerick** [comment]
If you're trying to build against 2.9, you can try this link until the fix for Gitpods changes are in place. 

https://gitpod.io/#https://github.com/edgetx/edgetx/tree/2.9-gitpod-workspace

**Reply by JustinKScott** [reply]
well, shoot.  That worked.   Thank you!
